docs/blueprints/EP000.rst

```diff
@@ -0,0 +1,153 @@
+:EP: 00
+:Title: Enhancement Proposals
+:Authors:
+    - Antonio Francisco <ajoaoff@gmail.com>
+    - Carlos Magno <cmagnobarbosa@gmail.com>
+    - Gleyberson Andrade <gleybersonandrade@gmail.com>
+    - Humberto Diógenes <hdiogenes@gmail.com>
+    - José Mauro Ribeiro <zemauror@gmail.com>
+    - Rogério Motitsuki <rogerio@ansp.br>
+:Created: 2020-05-14
+:Kytos-Version: 2020.1
+:Status: Draft
+:Type: Standards Track
+
+
+########
+Abstract
+########
+This blueprint is proposed to clarify what a blueprint is, its purpose, list the types of blueprints, explain which sections should be included in a blueprint, the information that should be included in each topic and the workflow involving a blueprint proposal. Therefore, this blueprint is intended to work as meta blueprint.
+
+##########
+Motivation
+##########
+Kytos Project uses word 'blueprint' to refer to a very specific type of document that plays a very important role in development process. Anyone who is not familiar with the concept should be able to understand and start a draft blueprint using the EP0 (blueprint 00).
+
+
+###################
+What is a blueprint
+###################
+A blueprint is a document used to specify the details of any process, features and anything else that is desirable to be implemented in Kytos Project and cannot be reduced to a Github issue or a meeting to be outlined.
```

Comment:
```suggestion
A blueprint is a document used to specify the details of any process, features, and anything else that is desirable to be implemented in the Kytos Project and cannot be reduced to a Github issue or a meeting to be outlined.
```

Reply:
This is called the "Oxford comma" and is completely optional, Mauro.

Reply:
Oh, good to know! Thanks guys.